docs/source/api_docs/expressions.rst

```diff
@@ -99,6 +99,7 @@ The following methods are available under the ``expr.str`` attribute.
    Expression.str.endswith
    Expression.str.concat
    Expression.str.split
+   Expression.str.extract
```

Comment:
A thought about docs: separate out the regex functions into their own section? Currently it's hard to tell which are regex and which are not.

Reply:
I think that's a good idea, but I think I'll finish up all the other regex expressions first, then see how they look all together in the docs.

Reply:
Yep, no need to do it in this PR, just something to keep in mind